### ISSUE TITLE
Incorrect Filtering of First Network in Hosts -> Networks Page After Update to v6.2.240814

### ISSUE THREAD

**Issue by edyatl** [ntop/ntopng#8628]
**Environment**:
* OS name: Ubuntu
* OS version: 22.04.4 LTS
* Architecture: amd64
* ntopng version/revision: ntopng Community v.6.2.240814 rev.24238
* Commit Reference: https://github.com/ntop/ntopng/commit/9a35ae52a1976b4df9c8eecd039dfac8ef7371ba
* Web Browser: Chromium Version 127.0.6533.119 (Official Build) Arch Linux (64-bit)

**What happened**:

After updating to ntopng Community v.6.2.240814 on Ubuntu 22.04, I encountered an issue in the web interface under `Hosts -> Networks`. When selecting the first network link in the table, the page loads without filtering the hosts by the selected network and instead shows all hosts. This issue only occurs with the first network in the list. Selecting any subsequent network works as expected, filtering hosts correctly.

**Expected Behavior:**

The page should display only the hosts associated with the selected network.

**How did you reproduce it?**:

1. Navigate to `Hosts -> Networks` in the ntopng web interface.
2. Click on the first network link in the table (which points to `/lua/hosts_stats.lua?network=0`).
3. Observe that the resulting page displays all hosts instead of filtering by the selected network.
4. Click on any other network in the list, and the filtering works as expected.

**Debug Information**:

- Additional information:
  - The issue did not occur in the previous version of ntopng.
  - The URLs generated for the first network selection appear to incorrectly process the `network=0` parameter, while other networks are processed correctly.
  - No related issues were found in the existing GitHub issues.

**Comment by MatteoBiscosi** [comment]
Hi @edyatl sorry but in my lab everything works correctly, could you please share a couple of screenshots?

**Comment by edyatl** [comment]
Hi @MatteoBiscosi,

Thank you for the quick response. I've created some screenshots to help illustrate the issue. Please find them attached below:

- **First Screenshot:** This shows the "Hosts -> Networks" table. When I click on the first network link (192.168.88.0/24), it redirects to an unfiltered page displaying all hosts, rather than filtering by the selected network.
![image](https://github.com/user-attachments/assets/d0a2e023-bd49-4ce8-875f-2875cd6ec906)

- **Second Screenshot:** After clicking on the first network link (192.168.88.0/24), it directs to the "Hosts -> Active" page. However, instead of showing only the hosts within the selected network, the page displays hosts from multiple networks, indicating the filtering is not applied.
![image](https://github.com/user-attachments/assets/e016e2a9-a0b1-40d5-9d84-6bf84cd72683)

- **Third Screenshot:** After clicking on the second network link (192.168.103.0/24), the "Hosts -> Active" page appears. This time, it correctly filters and displays only the hosts within the selected network, showing two active hosts as expected.
![image](https://github.com/user-attachments/assets/7dc37694-5648-434a-9e65-adc6d6b42de1)

It seems like the issue is specifically with the first network entry in the table. Please let me know if you need any more information or further troubleshooting steps on my end.

Thank you for your assistance!

**Comment by cardigliano** [comment]
@edyatl we pushed a patch that probably fixes this, please update and let us know

**Comment by edyatl** [comment]
Hi @cardigliano ,
I have updated to the latest version with `apt`.

Now running ntopng Community v.6.2.240820 rev.24271 (Ubuntu 22.04.4 LTS) with commit [11d405b](https://github.com/ntop/ntopng/commit/11d405b57530971c13f489634f0148b52ae1d6eb).

I can confirm that the issue with Incorrect Filtering of the First Network in the Hosts -> Networks Page has been resolved. The functionality is working as expected, and the filtering is now operating correctly as it did before the previous update.

Thank you for the quick patch!
